code/modules/mob/living/carbon/human/species_types/shadowpeople.dm

```diff
@@ -173,6 +173,7 @@
 	righthand_file = 'icons/mob/inhands/antag/changeling_righthand.dmi'
 	item_flags = ABSTRACT | NODROP | DROPDEL
 	w_class = WEIGHT_CLASS_HUGE
+	total_mass = 3.5

```

Comment:
On the other hand, this is acceptable since nightmares don't have access to any form of adrenals.